src/site/includes/appeals-banner.html

```diff
@@ -0,0 +1,15 @@
+  <div class="usa-alert-full-width usa-alert-full-width-warning">
+    <div aria-live="polite" role="alert" class="usa-alert usa-alert-warning">
```

Comment:
Let's change the `aria-live` to "assertive". I've seen several industry leaders mark their alert boxes up that way. I also think we could move the `aria-live` and `role` attributes to the outer container, unless it causes a CSS issue.

Reply:
made requested changes.